src/wiki/conf/settings.py

```diff
@@ -35,15 +35,15 @@
 #:    WIKI_MARKDOWN_KWARGS = {'extension_configs': {'toc': _('Contents of this article')}}
 MARKDOWN_KWARGS = {
     'extensions': [
-        'footnotes',
-        'attr_list',
-        'smart_strong',
-        'footnotes',
-        'attr_list',
-        'def_list',
-        'tables',
-        'abbr',
-        'sane_lists',
+        'markdown.extensions.footnotes',
+        'markdown.extensions.attr_list',
+        'markdown.extensions.smart_strong',
+        'markdown.extensions.footnotes',
+        'markdown.extensions.attr_list',
+        'markdown.extensions.def_list',
+        'markdown.extensions.tables',
+        'markdown.extensions.abbr',
+        'markdown.extensions.sane_lists',
     ],
     'extension_configs': {
```

Comment:
This is worth a note in the release notes since it means that if someone added `WIKI_MARKDOWN_KWARGS['extension_configs']`, they might have addressed the modules with just the module name and not the path.

Reply:
Added a note

Reply:
Awesome, thanks :+1: :)